File: Difficulty: Basic/Pattern 7/pattern-7.java
class Solution {

    void printTriangle(int n) {
        // code here
        for(int i=0;i<n;i++){
      
            for(int space=0;space<n-i-1;space++){
                System.out.print(" ");
            }
            for(int j=0;j<2*i+1;j++){
                System.out.print("*");
            }
            System.out.println();
        }
        }
    }

File: Difficulty: Basic/Pattern 7/pattern-7.py
#User function Template for python3

class Solution:
    def printTriangle(self, N):
        # Code here
        for i in range(1,N+1):
            for space in range(1,N-i+1):
                print(" ",end="")
            for j in range(2*i-1):
                print("*",end="")
            print() 